### ISSUE TITLE
'SerialBLE' was not declared in this scope

### ISSUE THREAD

**Issue by danic1977** [blynkkk/blynk-sketch-generator#95]
hi, i have this message error.
i have read the various solutions but its don't work anyway. I have followed the instruction to install libraries but when i upload the sketch to my arduino uno or mega error is the same. i use code from the site, codes example but its make the same error. how can i fix? 
i have windows 10 s.o.

**Comment by mcattani** [comment]
change: SoftwareSerial SwSerial(10, 11); // RX, TX
for: SoftwareSerial SerialBLE(10, 11); // RX, TX

**Comment by MasterNator** [comment]
I have learned this the hard way, I should of come here and read this error note! This took me days lol this is my very first analog read project, and went balls deep with the hc-05 ble blynk app- way too much to learn on on your own once starting out btw aha. Thought i was being super stupid but yes this should be fixed Mcattani! I was about to report this problem till I found this thread! Always check Google if you don't wanna figure it out I guess is my lesson learned today!
